backend-config/backend-config.go

```diff
@@ -71,7 +71,7 @@ type BackendConfig interface {
 	WaitForConfig(ctx context.Context)
 	Subscribe(ctx context.Context, topic Topic) pubsub.DataChannel
 	Stop()
-	StartWithIDs(ctx context.Context, workspaces string)
+	StartWithIDs(ctx context.Context, workspaces string, canUseCache bool)
```

Comment:
this looks too obtrusive

Reply:
So another way would be to read the config in the `StartWithIDs` function right? Or do you something else in mind?